src/main/java/calculator/Number.java

```diff
@@ -0,0 +1,35 @@
+package calculator;
+
+import java.util.Arrays;
+import java.util.stream.IntStream;
+
+/**
+ * @author Geonguk Han
+ * @since 2020-02-08
+ */
+public class Number {
```

Comment:
일급 컬렉션 👍 복수 명사를 클래스 이름으로 사용하면 컬렉션임을 더 쉽게 표현할 수 있겠죠?

```suggestion
public class Numbers {
```